pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -791,9 +791,32 @@ cdef class _Timedelta(timedelta):
     @property
     def nanoseconds(self):
         """
-        Number of nanoseconds (>= 0 and less than 1 microsecond).
+        Return the number of nanoseconds (n), where 0 <= n < 1 microsecond.
+       
+        Returns
+        -------
+        int
```

Comment:
shouldn't this be on the same line with a colon? @TomAugspurger @jorisvandenbossche

Reply:
This is fine as is. It's typically

```
name : type
    Explanation
```

but when the name is obvious (it'd just be `nanoseconds` here), it can be omitted.